src/main/resources/webapp/WEB-INF/views/data-man/chargepoints.jsp

```diff
@@ -48,7 +48,7 @@
             <tbody>
             <c:forEach items="${unknownList}" var="item">
                 <tr>
-                    <td>${item.key}</td>
+                    <td><encode:forHtml value="${item.key}" /></td>
```

Comment:
The same fix should be applied in some other places (not an exhaustive list):
https://github.com/steve-community/steve/blob/master/src/main/resources/webapp/WEB-INF/views/data-man/chargepointDetails.jsp#L89-L93
https://github.com/steve-community/steve/blob/master/src/main/resources/webapp/WEB-INF/views/data-man/chargepointDetails.jsp#L95-L96
https://github.com/steve-community/steve/blob/master/src/main/resources/webapp/WEB-INF/views/data-man/transactions.jsp#L122
https://github.com/steve-community/steve/blob/master/src/main/resources/webapp/WEB-INF/views/data-man/transactionDetails.jsp#L39
https://github.com/steve-community/steve/blob/master/src/main/resources/webapp/WEB-INF/views/data-man/transactionDetails.jsp#L62
https://github.com/steve-community/steve/blob/master/src/main/resources/webapp/WEB-INF/views/data-man/ocppTags.jsp#L51

Reply:
see https://github.com/steve-community/steve/pull/1527#issuecomment-2267115652